src/components/typography/Display.test.ts

```diff
@@ -0,0 +1,52 @@
+import { render, screen, cleanup } from "@testing-library/react-native";
+
+import Display from "./Display";
+
+afterEach(cleanup);
+
+describe("<Display />", () => {
+  it("should display the correct text", () => {
+    render(Display({ children: "test" }));
```

Comment:
I may be wrong, but I see more people rendering components like this and I guess it's better. You should take a look at better patterns for testing React Native applications.
```suggestion
    render(<Display children="test" />);
    // or I guess you can
    render(<Display>test</Display>)
```

Reply:
For some reason i does not work this way. I does not find the component.

Reply:
> For some reason i does not work this way. I does not find the component.

It should, I looked at some examples, try to use `.tsx` files or check your jest configuration

Reply:
I looked again in the expo configuration, it says if you are using typescript use .ts or .tsx files, I renamed the files to .tsx, it makes sense it was not working it is jsx syntax.